modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -935,6 +938,18 @@ public SimpleConfiguration getAppConfiguration() {
     return subset(APP_PREFIX);
   }
 
+  /**
+   * @return A subset of this configuration using the prefix {@value #SCAN_PREFIX}. Any change made
+   *         to subset will be reflected in this configuration, but with the prefix added. This
+   *         method is useful for setting application configuration before initialization. For
+   *         reading application configuration after initialization, see
+   *         {@link FluoClient#getAppConfiguration()} and {@link Context#getAppConfiguration()}
+   * @since 1.2
+   */
+  public SimpleConfiguration getScanConfiguration() {
```

Comment:
It occurred to me that the scan command is not part of Fluo's java API.   The scan command is outside of the Java API.  These configuration are related to Fluo's java API.  Based on this, I am thinking we should only have the command line options and not these configuration options.

It should be possible to use Fluo only via the Java API without ever using the Fluo tarball.  In this case, this scan configuration is out of place and confusing.

Is there a use case for having these config options that the command line options can not satisfy?

Reply:
I don't think so. I just think this could be convenient. 
But you are right. I'll remove this implementation.

Reply:
I am OK with the configuration options but I agree with that they shouldn't be in FluoConfiguration or `fluo-app.properties`.  I think all configuration in `fluo-app.properties` will get loaded to Zookeeper and setting them would affect everyone that runs the `fluo scan` command for that application from anywhere on the cluster. Maybe the `fluo scan` could have an command line option for a path to a properties file where this configuration could be specified.